src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
Inheritance here is not a good solution, because with inheritance relations between u classes should be 'is', but Lottery is. not CollorSupplier, I would recommend use Composition here, it's a 'has-a' relationship between classes.